llmfoundry/data/text_data.py

```diff
@@ -88,6 +89,7 @@ def __init__(self,
                  shuffle_algo: str = 'py1b',
                  shuffle_seed: int = 9176,
                  shuffle_block_size: int = 1 << 18,
+                 sampling_method: str = 'balanced',
```

Comment:
just to double check, this is the same as the default in streaming, right?

Reply:
yep same default!